env0/resource_environment.go

```diff
@@ -380,6 +386,12 @@ func setEnvironmentSchema(ctx context.Context, d *schema.ResourceData, environme
 		return fmt.Errorf("schema resource data serialization failed: %v", err)
 	}
 
+	if val := d.Get("vcs_pr_comments_enabled").(bool); !val && environment.VcsPrCommentsEnabled {
```

Comment:
this is important. Although the default is 'false' it's possible vcs_pr_comments_enabled is set to 'true' (E.g. alias was used).
To avoid drifts - it's ignored in this case.